plugins/jobs/server/models/job.py

```diff
@@ -55,8 +56,20 @@ def _validateStatus(self, status):
             raise ValidationException(
                 'Invalid job status %s.' % status, field='status')
 
+    def _validateParent(self, parentJobId):
+        try:
+            self.model('job', 'jobs').load(parentJobId, force=True, exc=True)
+        except ValidationException:
+            raise ValidationException('%s is not a valid Job Id' % parentJobId)
```

Comment:
Is there a good reason to catch and re-raise with a different error message? The default one is "No such job: [id]", which seems just as good to me.